solr/conf/solrconfig.xml

```diff
@@ -48,7 +48,7 @@
           alternative_title_tsim
           active_fedora_model_ssi
           title_tsim
-          author_tsim
+          author_tsimesim
```

Comment:
was this intentional?

Reply:
Looks  like we must have propagated it from Blacklight into some gems before https://github.com/projectblacklight/blacklight/commit/7899e819cbfe18adbeac9cb2baea6ad4dedb85b1 landed.